pkg/apis/kudo/v1beta1/operatorversion_types_helpers.go

```diff
@@ -0,0 +1,13 @@
+package v1beta1
+
+import (
+	"fmt"
+)
+
+func OperatorInstanceName(operatorName string) string {
+	return fmt.Sprintf("%s-instance", operatorName)
+}
+
+func OperatorVersionName(operatorName, operatorVersion string) string {
```

Comment:
More nitpicking: Isn't this and `OperatorInstanceName` better in `operator_types_helpers` because it extracts information from operator names? I mistook `operatorVersion` for and `OperatorVersion`, that's why I suggested that this should be in `operatorversion_types_helpers`... but it's in fact a _version version_ :). So let's just call this parameter `version` so that we don't mistake it with an `OperatorVersion`.